### PR TITLE
fix: update db-types and add documentation

### LINKED ISSUE #1580: NFT API Db-Types autogenerates types
Upload stats API types were hand-typed, they are autogenerated by `yarn db-types` however, this is unnecessary and is causing linter issues.

Thanks to @hugomrdias for finding and pointing this out!

### PR DESCRIPTION
**Pull request by dashcraft**
Fixes #1580 
Re-ran db-types and added documentation for the db-types command for posterity.

### CODE REVIEW

**Comment by alanshaw** [comment]
@dashcraft is this still relevant? Is it ready for review?

**Comment by dashcraft** [comment]
@alanshaw yeah it's just running the automated db-types, i had hand typed some and this uses the automated tool to generate the correct ones.

It's been hanging out in pending review for awhile

File: packages/api/README.md [hunk 1]
```diff
@@ -79,6 +79,15 @@ When prompted for a value enter one of the following permission combinations:
 - `r-` = read only mode
 - `rw` = read and write (normal operation)
 
+## DB Types
+
+The postgres rest api can generate automatic type defs based on the table and column
+names. To use this, make sure you've spun down your nft dev environment and run
+`yarn db-types` from packages/api root directory. This will auto generate the
+`packages/api/utils/db-types/d.ts` file.
```
Comment (alanshaw) [inline packages/api/README.md:87]:
```suggestion
`packages/api/utils/db-types.d.ts` file.
```

**Review by alanshaw — APPROVED** [review]
This LGTM, just one small typo. Could you please rebase before merging (and re-run the type generation script) as it has been a while since this was opened!

**Pull request merged**